cmd/create/cluster/cmd.go

```diff
@@ -601,6 +604,12 @@ func init() {
 		false,
 		"Technology Preview: Enable the use of Hosted Control Planes",
 	)
+	flags.MarkHidden("hosted-cp")
```

Comment:
Is this related to this change? I dont think it is needed?

Reply:
Likely a mistake during a rebase, will remove it. Thanks.